package.json

```diff
@@ -312,6 +312,7 @@
     "@kbn/core-elasticsearch-client-server-internal": "link:src/core/packages/elasticsearch/client-server-internal",
     "@kbn/core-elasticsearch-server": "link:src/core/packages/elasticsearch/server",
     "@kbn/core-elasticsearch-server-internal": "link:src/core/packages/elasticsearch/server-internal",
+    "@kbn/core-elasticsearch-server-utils": "link:src/core/packages/elasticsearch/server-utils",
```

Comment:
I had to introduce a new package to avoid circular dependencies: 
server → client-server-mocks → client-server-internal → server